app/assets/stylesheets/config/_layout.scss

```diff
@@ -29,4 +34,4 @@
     transform: translateX(-50%);
     margin-left: -($rating-widget-width/2);
```

Comment:
We probably should rewrite it to use flexbox (to avoid such hacks), but I guess now's not the time yet :(